spillr_paper.Rmd

```diff
@@ -662,11 +668,11 @@ of the isotope) and La139Di, and similarly for the other rows. On the horizontal
 
 In all six panels (A--F), we observe that `spillR` compensates most strongly in the low counts. In panel C, CD3 (Yb173Di) against HLA-ABC (Yb172Di), `CATALYST` can be seen to compensate strongly in the middle range. It removes the spherical pattern that shows correlation between the two markers. `spillR` preserves this correlation structure and only masks out the lower counts of CD3 (Yb173Di). This highlights a key difference between `spillR` and `CATALYST`: `spillR` does not correct all counts by shrinking them, but rather removes some counts, following the idea that the distribution of the remaining counts is close to the true distribution. `CATALYST` follows another strategy by shrinking counts across the entire range. 
 \textcolor{red}{
-\texttt{spillR} (fast) compensates most aggressively, removing everything that is not in the support of the beads distributions.
+\texttt{spillR} (naive) compensates most aggressively, removing everything that is not in the support of the beads distributions.
 }
 
 \textcolor{red}{
-The panels D and F show no compensation by \texttt{CATALYST}. For the same panels our \texttt{spillR} methods compensate strongly. Our diagnostic plots using function \texttt{plotDiagnostics} in \texttt{spillR} give some indication on why our method compensates that strongly: The original experiment is designed such that we expect no spillover in the channels on the vertical axis, but our diagnostic plots show that the bead distribution of at least one spillover marker---as identified by the spillover matrix in \texttt{CATALYST}---overlap with the first mode in the distribution of real cells. Thus, the strong compensation by \texttt{spillR} is consistent with our assumption (A1). \texttt{spillR} (fast) corroborates our findings as it completely removes all counts in that area, which occurs when there is only spillover and no signal from the target marker.
+The panels D and F show no compensation by \texttt{CATALYST}. For the same panels our \texttt{spillR} methods compensate strongly. Our diagnostic plots using function \texttt{plotDiagnostics} in \texttt{spillR} give some indication on why our method compensates that strongly: The original experiment is designed such that we expect no spillover in the channels on the vertical axis, but our diagnostic plots show that the bead distribution of at least one spillover marker---as identified by the spillover matrix in \texttt{CATALYST}---overlap with the first mode in the distribution of real cells. Thus, the strong compensation by \texttt{spillR} is consistent with our assumption (A1). \texttt{spillR} (naive) corroborates our findings as it completely removes all counts in that area, which occurs when there is only spillover and no signal from the target marker.
```

Comment:
...removes all counts in that _range_, which occurs when in the beads experiments only spillover and no signal from the target marker is observed in that range.

Reply:
I would try to avoid `remove` at all costs, especially now that we set it to zero

Reply:
I changed "area" to "range" and rephrased the explanation as to when this happens.
Resolve if you agree.